daphne_worker_test/tests/test_runner.rs

```diff
@@ -59,8 +63,9 @@ const LEADER_TASK_LIST: &str = r#"{
         "vdaf_verify_key": "1fd8d30dc0e0b7ac81f0050fcab0782d"
     },
     "410d5e0abd94a88b8435a192cc458cc1667da2989827584cbf8a591626d5a61f": {
-        "leader_url": "http://leader:8787",
-        "helper_url": "http://127.0.0.1:9788",
+        "version": "v01",
```

Comment:
Given that we require the version in the URL, it's a bit awkward to have a separate version as well. Let's remove this field from the TaskConfig and the corresponding version checks. Right now they're fairly cumbersome and don't add a lot of value.